components/lib/VButton.vue

```diff
@@ -117,7 +117,7 @@ button:disabled {
 
 /* Primary Button Styles */
 button[data-variant='primary'] {
-  background: linear-gradient(0deg, #0085ff 0%, #29c8fa 100%);
+  background: linear-gradient(180deg, #0085ff -4.5%, #29c8fa 100.1%);
```

Comment:
can we use custom css var for color here to maintain consistency

Reply:
@karthik-durai  yes we can do that but currently we have these values relevant only to button and are not reused anywhere. Hence we didn't keep it as a CSS variable

Reply:
but consider using CSS var even we use the color in one place.